index.js

```diff
@@ -0,0 +1,116 @@
+/*Iteration 1: Names and Input
+1.1 Create a variable hacker1 with the driver's name.*/
+let hacker1 = "Juergen";
+
+
+//1.2 Print "The driver's name is XXXX".//
+
+console.log(`The driver´s name is ${hacker1}`);
+
+//1.3 Create a variable hacker2 with the navigator's name.//
+
+let hacker2 = "AlinaSergej";
+/*Iteration 1: Names and Input
+1.1 Create a variable hacker1 with the driver's name.*/
+let hacker1 = "Juergen";
+
+
+//1.2 Print "The driver's name is XXXX".//
+
+console.log(`The driver´s name is ${hacker1}`);
+
+//1.3 Create a variable hacker2 with the navigator's name.//
+
+let hacker2 = "AlinaSergej";
+
+//1.4 Print "The navigator's name is YYYY".//
+
+console.log(`The navigator´s name is ${hacker2}`);
+
+/*Iteration 2: Conditionals
+2.1. Depending on which name is longer, print: - The Driver has the longest name, it has XX characters or - Yo, navigator got the longest name, it has XX characters or - Wow, you both got equally long names, XX characters!*/
+
+if (hacker1.length >=hacker2.length){
```

Comment:
could be greatly simplified by extracting the condition as an expression to use in the if else branching